app/screens/reader-screen/reader-screen.tsx

```diff
@@ -31,6 +33,8 @@ export class ReaderScreen extends React.Component<ReaderScreenProps> {
 
   private onPressContentItem = (id: string) => {
     const content = this.props.readerStore.contents.get(id)
+    logAnalyticsEvent('select_content', { contentType: 'content', itemId: id })
+    // logAnalyticsEvent('OpenArticle', { url: id })
```

Comment:
Can delete L37?

Reply:
This was for a TODO to add a filter to trim every payload to 100 char, because of firebase payload limitation
Will add comment/just implement that function